src/main/java/org/springframework/cloud/deployer/scheduler/spi/kubernetes/EntryPointStyle.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.scheduler.spi.kubernetes;
+
+import java.util.EnumSet;
+
+import org.springframework.cloud.deployer.scheduler.spi.kubernetes.support.RelaxedNames;
+
+/**
+ * Defines container entry point styles that are available. The selected entry point style
+ * will determine how application properties are made available to the container.
+ *
+ * @author Chris Schaefer
+ */
+public enum EntryPointStyle {
```

Comment:
we should probably use the existing `EntryPointStyle` rather than duping here as they will have the same options